BraveWallet/Extensions/BraveWalletSwiftUIExtensions.swift

```diff
@@ -146,3 +146,19 @@ extension BraveWallet {
   /// The address that is expected when you are swapping ETH via SwapService APIs
   public static let ethSwapAddress: String = "0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee"
 }
+
+extension URL {
+  /// The origin of the URL, with bold eTLD+1 when available.
+  var originWithEtldPlusOne: Text {
+    let origin = absoluteString
```

Comment:
This was originally an extension on `URLOrigin` but was updated to `URL` to be more versatile. I think with above comment it might make more sense to setup a `OriginText` view that accepts `URLOrigin` in the constructor; it looks like we have `URLOrigin` everywhere we might need this.

Reply:
Fixed in [c5a136b](https://github.com/brave/brave-ios/pull/5187/commits/c5a136bcc97febb55324ba1971ead39050b455eb)